build/hydra-proxy.nix

```diff
@@ -1,33 +1,26 @@
 {
   config,
-  lib,
   pkgs,
   ...
 }:
 
-let
-  bannedUserAgentPatterns = [
-    "Trident/"
-    "Android\\s[123456789]\\."
-    "iPod"
-    "iPad\\sOS\\s"
-    "iPhone\\sOS\\s[23456789]"
-    "Opera/[89]"
-    "(Chrome|CriOS)/(\\d\\d?\\.|1[01]|12[4])"
-    "(Firefox|FxiOS)/(\\d\\d?\\.|1[01]|12[012345679]\\.)"
-    "PPC\\sMac\\sOS"
-    "Windows\\sCE"
-    "Windows\\s95"
-    "Windows\\s98"
-    "Windows\\sNT\\s[12345]\\."
-  ];
-in
 {
   networking.firewall.allowedTCPPorts = [
     80
     443
+    9001
```

Comment:
Shouldn't we limit that to Prometheus and not make it public?

Reply:
Eventually, but up until now all exporters have been publically accessible.

Reply:
That is not that great.

Depending on the exporter this might be more or less of a problem. Some just expose internal counters but others execute commands on scrape and I am not sure if they have rate limit and if you could cause damage by spamming those endpoints.